libs/designsystem/src/lib/components/progress-circle/progress-circle-ring.component.spec.ts

```diff
@@ -106,7 +106,7 @@ describe('ProgressCircleRingComponent', () => {
 
     it('should render background stroke with the defined stroke width', () => {
       spectator.detectChanges();
-      expect(spectator.query('circle.progress')).toHaveAttribute(
+      expect(spectator.query('circle.circle')).toHaveAttribute(
```

Comment:
Just out of curiosity: why change to `circle.circle` instead of `circle.progress`? Won't it be testing something else than it was before? I haven't looked too much into it, so I'm mostly asking to get cleverer 🧠

Reply:
This test, and the one above it, was completely identical. So now it should actually test what we expect 😃